api/conf/conf.yaml

```diff
@@ -15,12 +15,21 @@
 # limitations under the License.
 #
 
+# yamllint disable rule:comments-indentation
 conf:
-  listen:                 # yamllint disable rule:comments-indentation
+  listen:
     # host: 127.0.0.1     # the address on which the `Manager API` should listen.
                           # The default value is 0.0.0.0, if want to specify, please enable it.
                           # This value accepts IPv4, IPv6, and hostname.
     port: 9000            # The port on which the `Manager API` should listen.
+
+  # ssl:
+  #   host: 127.0.0.1     # the address on which the `Manager API` should listen for HTTPS.
+                          # The default value is 0.0.0.0, if want to specify, please enable it.
```

Comment:
We may also support the mTLS.

Reply:
sure. we may support mTLS in the future.